apps/AEPSampleAppNewArchEnabled/android/app/src/main/java/com/anonymous/AEPSampleAppNewArchEnabled/MainActivity.kt

```diff
@@ -0,0 +1,61 @@
+package com.anonymous.AEPSampleAppNewArchEnabled
```

Comment:
Could we replace `anonymous` with a meaningful name?

Reply:
have updated name of the package